src/main/java/org/wso2/carbon/inbound/feedep/FeedInject.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2006, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We need not to change copyright year. the year at copyright means the first date it is established.

Reply:
copyright year was already wrong. Because the connector was released on 2016. So I changed